nixos/modules/services/backup/borgbackup.xml

```diff
@@ -117,18 +117,27 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAID78zmOyA+5uPG4Ot0hfAy+sLDPU1L4AiIoRYEIVbbQ/
   };
 };</programlisting>
   </para>
-  <para>The following few commands (run as root) let you test your backup.
-      <programlisting>
-> nixos-rebuild switch
+  <section xml:id="opt-services-backup-borgbackup-testing">
+   <title>Testing remote backups</title>
+   <para>Every job has a corresponding wrapper script installed into <xref linkend="opt-environment.systemPackages"/>. Inspect the wrapper variables by reading <filename>/run/current-system/sw/bin/borg-job-&lt;NAME&gt;</filename>. This script can be used to test backups or manage your remote repo.
+   </para>
+   <para>The following few commands (run as root) show how to test your backup.
+<screen>
+<prompt># </prompt> nixos-rebuild switch
 ...restarting the following units: polkit.service
-> systemctl restart borgbackup-job-backupToLocalServer
-> sleep 10
-> systemctl restart borgbackup-job-backupToLocalServer
-> export BORG_PASSPHRASE=topSecrect
-> borg list --rsh='ssh -i /run/keys/id_ed25519_my_borg_repo' borg@nixos:.
+<prompt># </prompt> systemctl restart borgbackup-job-backupToLocalServer
```

Comment:
```suggestion
<prompt># </prompt> systemctl start borgbackup-job-backupToLocalServer
```
Why restart?

Reply:
No idea - that's how it was before. `systemctl start` would be fine.